IM_calculation/scripts/calculate_ims.py

```diff
@@ -196,11 +196,10 @@ def main():
 
     im_options = {}
 
-    if "pSA" in im:
-        im_options["pSA"] = calc.validate_period(args.period, args.extended_period)
-
-    if "SDI" in im:
-        im_options["SDI"] = calc.validate_period(args.period, args.extended_period)
+    valid_periods = calc.validate_period(args.period, args.extended_period)
+    for periodic_im in ["pSA", "SDI"]:
```

Comment:
Arguably without the for loop it's more clear. I do like the support variable though. Happy as is.

Reply:
Sure. Agreed and fixed.